docs/ReactiveExtensions.md

```diff
@@ -0,0 +1,55 @@
+ReactiveCocoa
+=============
+
+Moya providers an optional `ReactiveMoyaProvider` subclass of
+`MoyaProvider` that does a few interesting things. Instead of
+calling the `request()` method and providing a callback closure
+to be executed when the request completes. Instead, we use
```

Comment:
Too many "instead"s, I propose a change to 

>Instead of calling the `request()` method and providing a callback closure to be executed when the request completes, we emit `RACSignal`s.